src/sage/rings/polynomial/polynomial_zmod_flint.pyx

```diff
@@ -483,6 +484,48 @@ cdef class Polynomial_zmod_flint(Polynomial_template):
 
     _mul_short_opposite = _mul_trunc_opposite
 
+    cpdef Polynomial _powmod_bigexp(self, exp, Polynomial_zmod_flint m):
```

Comment:
This will need a doctest.

Reply:
The doctests are in __pow__. Does it make sense to add a doctest here ? Or only for sanity checks ?

Reply:
It is policy that every method has a doctest. Perhaps move those doctests to here? Even better if you can isolate it with an explicit call.

Reply:
Okay, i ported basic tests from `__pow__`